tests/size_test.py

```diff
@@ -194,6 +196,10 @@ def testScientificNotation(self):
         self.assertEqual(size.parseSpec("1E-4KB"), Decimal("0.1"))
         self.assertEqual(Size("1E-10KB"), Size(0))
 
+    def testPickling(self):
+        s = Size("10 MiB")
+        self.assertEqual(s, cPickle.loads(cPickle.dumps(s)))
```

Comment:
But we care about pickling working not our implementation of some method working as it is written. Maybe not so unit-testy, but what @vojtechtrefny has in the test is what we actually want to test.